kroxylicious/src/main/java/io/kroxylicious/proxy/internal/net/EndpointRegistry.java

```diff
@@ -134,9 +135,10 @@ private record VirtualClusterRecord(CompletionStage<Endpoint> registrationStage,
             Objects.requireNonNull(deregistrationStage);
         }
 
-    private static VirtualClusterRecord create(CompletionStage<Endpoint> stage) {
-        return new VirtualClusterRecord(stage, new AtomicReference<>(), new AtomicReference<>());
-    }}
+        private static VirtualClusterRecord create(CompletionStage<Endpoint> stage) {
+            return new VirtualClusterRecord(stage, new AtomicReference<>(), new AtomicReference<>());
+        }
+    }
```

Comment:
This is a good example of actually getting the formatting right within a record that this change brings.